audit.py

```diff
@@ -178,20 +295,111 @@ def output_json(json_data: list, output: Optional[typer.FileTextWrite]) -> None:
     typer.echo(json.dumps(json_data, indent="  "), file=output)
 
 
-def output_text_elements(title: str, table: Table, started: datetime, output: Optional[typer.FileTextWrite]) -> None:
-    """
-    Output the text elements
-    :param title: title above the table
-    :param table: data to report
-    :param started: when creating this report started
-    :param output: output file to write to (default: stdout)
-    """
-    console = Console(file=output)
-    console.print(title)
-    console.print()
-    console.print(table)
-    console.print()
-    console.print(format_generated_timestamp(started))
+###
+# APPLICATION COMMANDS
+
+g = Github(config["github.com"]["token"])
+app = typer.Typer()
+
+
+class RepoTextReport(TextReportBase):
+
+    def __init__(self, started, output, include_archived_repositories, include_forked_repositories):
+        super().__init__(started, output)
+        self.include_archived_repositories = include_archived_repositories
+        self.include_forked_repositories = include_forked_repositories
+
+    def _table_header(self):
+        if self.table is None:
+            raise RuntimeError("not building a table")
+        self.table.add_column("Name")
+        if self.include_archived_repositories:
+            self.table.add_column("Archived", justify="center")
+        if self.include_forked_repositories:
+            self.table.add_column("Fork", justify="center")
+        self.table.add_column("Pushed at")
+        self.table.add_column("Pull request title")
+        self.table.add_column("Created at")
+
+    def _table_row(
+            self,
+            repo_name=None, archived=None, fork=None, pushed_at=None, title=None, created_at=None,
+            rowspan=0, first=False
+    ):
+        if self.table is None:
+            raise RuntimeError("not building a table")
+        row = []
+        row.append(repo_name)
```

Comment:
```suggestion
        row = [repo_name]
```

Reply:
Breaks the appearance of the systematic filling of `row` - not my taste.